pkgs/modules/rust/default.nix

```diff
@@ -1,18 +1,28 @@
-{ pkgs, lib, ... }:
+{ pkgs, ... }:
 let
-  inherit (pkgs.fenix.stable) toolchain;
+  toolchain = pkgs.fenix.stable;
```

Comment:
it might be confusing to name this toolchain when we later call toolchain.toolchain

Can we call this rust_stable or something? Not blocking since you know this ecosystem better than me.